### PR TITLE
Change BSC filter in hpc mon

### PR DESCRIPTION
**Pull request by mrceyhun**
BSC filter is changed to (OR cond):
```
((col('Site').endswith('_ES_PIC_BSC')) & (col('MachineAttrCMSSubSiteName0') == 'PIC-BSC')) | 
((col('Site').endswith('_ES_PIC')) & (col('MachineAttrCMSSubSiteName0') == 'PIC-BSC')) |  
```

`iterative_ndays_ago` param is added to easily modify previous n days of monitoring data. Always use it after reading its documentation.

### CODE REVIEW

**Comment by leggerf** [comment]
@mrceyhun are you waiting for Antonio confirmation before merging?

**Comment by mrceyhun** [comment]
@leggerf No, I compared it with old data and everything seems fine on the plots.

There is a small bug in Plotly plots and I was trying to implement to show total of each bar at the top which I could not achieve. Thats why I put `do-not-merge`. But this can be separated, removing do-not-merge.

**Review by leggerf — APPROVED** [review]
I'm approving, up to you to decide when to merge

**Pull request merged**
